release.json

```diff
@@ -10,6 +10,17 @@
 		"OMNIBUS_RUBY_BRANCH": "datadog-5.5.0",
 		"OMNIBUS_SOFTWARE_BRANCH": "master"
 	},
+	"5.22.0-rc.1": {
+		"AGENT_VERSION": "5.22.0-rc.1",
+		"AGENT_BRANCH": "5.22.0-rc.1",
+		"AGENT6_BRANCH": "6.0.0-beta.3",
```

Comment:
not sure this parameter is useful, but we should probably set it to the latest beta or `6.0.0-beta.7` like 5.21.1